src/pages/extensions/iter8/Iter8ExperimentList/FiltersAndSorts.ts

```diff
@@ -36,113 +29,51 @@ export const sortFields: SortField<Iter8Experiment>[] = [
     id: 'baseline',
     title: 'Baseline',
     isNumeric: false,
-    param: 'is',
+    param: 'is', // !! same as phase?
```

Comment:
I haven't tested yet the iter8 part, but it seems like there's two bugs spotted:
- this 'is' param is used for three different sortable fields, which means it probably won't work as expected with the URL params
- below, there's a "candidateFilter" but it is not tied to any filtering process, so I guess it won't work?

cc @lucasponce 
I would need to see how to setup iter8 for testing.

Reply:
I'll help on that, this PR would need a lot of detailed review as it may have side effects.